lib/graph/helpers/ui5Framework.js

```diff
@@ -363,13 +364,16 @@ export default {
 			});
 		}
 
+		const config = await Configuration.fromFile();
+
 		// Note: version might be undefined here and the Resolver will throw an error when calling
 		// #install and it can't be resolved via the provided library metadata
 		const resolver = new Resolver({
 			cwd: rootProject.getRootPath(),
 			version,
 			providedLibraryMetadata,
-			cacheMode
+			cacheMode,
+			ui5HomeDir: config.getUi5DataDir()
```

Comment:
so now, we have the config `ui5-data-dir` but we map it directly to `ui5-home-dir`. Is this intended?